src/components/item/ItemForm.vue

```diff
@@ -310,6 +337,17 @@ export default {
         </TogglingBlock>
       </TogglingBlock>
     </div>
+    <div
+      v-if="$route.name !== 'item'"
+      class="single-item-link"
+    >
+      <CustomLink
+        :to="{ name: 'item', params: { id: edittingItemObj.id } }"
+        title="open item separately"
+        target="_blank"
```

Comment:
I would like to add that external link to every link with target="_blank"
and we could even this attribute to use in css for that